linux_files/setup

```diff
@@ -410,19 +460,35 @@ fi
 }
 
 function explorerintegration {
-read -r -p "Would you like to enable Windows Explorer shell integration? [y/N] " response     
-if [[ "$response" =~ ^[yY]$ ]]
-then
+ read -r -p "Would you like to enable Windows Explorer shell integration? [y/N] " response     
+ if [[ "$response" =~ ^[yY]$ ]]
+  then
     echo "Enabling Windows Explorer shell integration."
+    cat << 'EOF' >> Install.reg
```

Comment:
Is this safe to use? .reg file suppose to be CRLF but `cat` would produce only LF